portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase

```diff
@@ -1406,23 +1418,16 @@ definition {
 			ACUtils.setItemName(itemName = "Dynamic Segment Test");
 
 			ACSegments.addSegmentField(segmentField = "Viewed Page");
-
-			ACSegments.editWebBehaviorCriterion(
-				indexField = "1",
-				occurenceNumber = "4",
-				searchTerm = "Blogs AC Title - Site Name - Liferay DXP");
 		}
 
 		task ("Check for data source origin") {
 			var key_indexField = "1";
 
 			Click(locator1 = "ACSegments#SELECT_BUTTON");
 
-			ACSegments.searchSegment(findSegment = "${documentTitle}");
+			ACSegments.searchSegment(findSegment = "Blogs AC Title - Site Name - Liferay DXP");
```

Comment:
Hey @joshitagaki, I think the name of the Data Source is different in the CI. I think you should use the `${dataSourceName}` variable in that case. What do you think?

`ACSegments.searchSegment(findSegment = "Blogs AC Title - Site Name - ${dataSourceName}");`

Reply:
Hey @ViniLopes20, so it seems to be that in CI the data source name will continue to be Liferay DXP. Using `${dataSourceName}` shouldn't be a problem though so I can just change it to use that since it would return the same thing.

Reply:
I already noticed that the name stays the same when you run the CI in your PR, but in the daily routine this would be a problem, I don't know the reason for this difference but I've noticed it hahaha. Thanks for changing